docs/storefront/stencil/themes/context/object-reference/schemas/settings.yml

```diff
@@ -77,6 +78,7 @@ properties:
   is_eu_ip_address:
     type: boolean
     description: A boolean value. Return true if the shoppers ip address is in the EU.
```

Comment:
Alternatives to this deprecated field are solution like https://ipinfo.io/products/ip-geolocation-api, https://www.bigdatacloud.com/ip-geolocation and others. While those solution are not free, they offer a generous free tier and offer better quality of the data and more data points than just the region.